ign_rviz_plugins/package.xml

```diff
@@ -20,14 +20,18 @@
   <depend>urdf</depend>
   <depend>visualization_msgs</depend>
 
-  <!-- Edifice (default) -->
-  <depend condition="$IGNITION_VERSION != 'dome'">ignition-common4</depend>
-  <depend condition="$IGNITION_VERSION != 'dome'">ignition-gui5</depend>
-  <depend condition="$IGNITION_VERSION != 'dome'">ignition-rendering5</depend>
-  <!-- Dome -->
-  <depend condition="$IGNITION_VERSION == dome">ignition-common3</depend>
-  <depend condition="$IGNITION_VERSION == dome">ignition-gui4</depend>
-  <depend condition="$IGNITION_VERSION == dome">ignition-rendering4</depend>
+  <!-- Fortress (default) -->
+  <depend condition="$IGNITION_VERSION == 'fortress'">ignition-common4</depend>
+  <depend condition="$IGNITION_VERSION == 'fortress'">ignition-gui6</depend>
+  <depend condition="$IGNITION_VERSION == 'fortress'">ignition-rendering6</depend>
+  <depend condition="$IGNITION_VERSION == ''">ignition-common4</depend>
+  <depend condition="$IGNITION_VERSION == ''">ignition-gui6</depend>
+  <depend condition="$IGNITION_VERSION == ''">ignition-rendering6</depend>
+
+  <!-- Edifice -->
+  <depend condition="$IGNITION_VERSION == edifice">ignition-common4</depend>
```

Comment:
Nit: `common4` is the same for edifice and fortress, so it could be depended unconditionally